etc/DependencyInstaller.sh

```diff
@@ -36,9 +36,10 @@ _installCommonDev() {
 
     # boost
     cd "${baseDir}"
-    wget https://github.com/boostorg/boost/archive/refs/tags/boost-${boostVersion}.tar.gz -o boost-${boostVersion}
-    tar -xf boost-${boostVersion}.tar.gz
-    cd boost-${boostVersion}
+    boostVersionUnderscore=${boostVersion//./_}
+    wget https://boostorg.jfrog.io/artifactory/main/release/${boostVersion}/source/boost_${boostVersionUnderscore}.tar.gz
```

Comment:
We should perform a hash check on the downloaded tar to prevent supply chain attacks.

Reply:
fixed on 495a716